spanner/api/Spanner.Samples/QuickStartAsync.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START spanner_quickstart]
```

Comment:
This sample needs to be different to others. It needs to be a full script that someone can just go `dotnet run` on so we need to have a different setup. I personally wouldn't migrate this at all and would leave it the way it is.

Reply:
kept old sample as it is.

Reply:
So maybe remove this file from the PR then and the test? Or am I missing something?

Reply:
As we have separate project for QuickStart removed it from here.